src/util/clone.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @function
+ * @description Deep clone a class instance.
+ * @param instance The class instance you want to clone.
+ * @returns A new cloned instance.
+ */
+export function clone<O>(instance: O): O {
```

Comment:
```suggestion
export default function clone<O>(instance: O): O {
```

this allows you to import it without the {} notation